tests/test_to_compressed_netcdf.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+import xarray as xr
+
+import bitinformation_pipeline as bp
+
+
+def test_to_compressed_netcdf():
+    """Test bm.get_bitinformation."""
+    label = "rasm"
+    ds = xr.tutorial.load_dataset(label)
+    # bitinformation_pipeline
+    bitinfo = bp.get_bitinformation(ds, dim=3)
+    keepbits = bp.get_keepbits(ds, bitinfo)
+    ds_bitrounded = bp.xr_bitround(ds, keepbits)
+    # save
+    ds.to_netcdf(f"{label}.nc")
+    ds.to_compressed_netcdf(f"{label}_compressed.nc")
```

Comment:
what do you think about this API? @observingClouds  full example end to end here.